clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerAtomicOperation.groovy

```diff
@@ -154,4 +158,30 @@ class UpsertAmazonLoadBalancerAtomicOperation implements AtomicOperation<UpsertA
     task.updateStatus BASE_PHASE, "Done deploying load balancers."
     operationResult
   }
+
+  private void updateApplicationSecurityGroupIngressRules(Map<String, String> securityGroupNamesToIds,
+                                                          ArrayList<Listener> listeners,
+                                                          RegionScopedProviderFactory.RegionScopedProvider regionScopedProvider) {
+    Names names = Names.parseName(description.getName())
+    String application = names.getApp()
+    try {
+      if (application in securityGroupNamesToIds) {
+        String applicationSecurityGroup = securityGroupNamesToIds[application]
+        task.updateStatus BASE_PHASE, "Configuring ingress rules for application security group $applicationSecurityGroup ..."
+        List<IpPermission> permissions = listeners.collect {
+          new IpPermission(
+            ipProtocol: 'tcp',
+            fromPort: it.getInstancePort(),
+            toPort: it.getInstancePort(),
+            ipv4Ranges: Collections.singletonList(new IpRange(cidrIp: "0.0.0.0/0"))
+          )
+        }
+
+        regionScopedProvider.securityGroupService.authorizeSecurityGroup(applicationSecurityGroup, permissions)
+      }
+    } catch (ignored) {
```

Comment:
Let's avoid ignoring exceptions. If you want to catch everything, do `Throwable t` and log it as an error, in addition to the status update.